master/buildbot/process/buildrequestdistributor.py

```diff
@@ -1173,6 +1173,9 @@ def _selectNextBuildRequest(self, queue, asyncFunc):
 
     @defer.inlineCallbacks
     def _procesBuildRequestsActivityLoop(self):
```

Comment:
@campos-ddc this is called whenever the masters needs to run a new build or check if new buildrequests can get processed, examples: new buildrequest, new slaves attached, I think is fine not to sleep.